src/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -134,7 +139,12 @@ export const OcmClusterDetailsFormFields = ({
           {values.cpuArchitecture}
         </StaticTextField>
       ) : (
-        <CpuArchitectureDropdown />
+        cpuArchitectures && (
```

Comment:
What if the data fails to load? It wouldn't be possible to create any cluster. Maybe the `CpuArchitectureDropdown` can handle the case and return the list with x86 only.

Reply:
True. I'll add only x86.